### DISCUSSION TITLE
HCI Problem : code: 17, error: Invalid Index

### DISCUSSION THREAD

**Discussion by Ether-YiTseWu** [hbldh/bleak#956] [Resolved discussions] [answered]
### Description

My Error is : ***[code: 17, error: Invalid Index]***

### What I Did

I use ***hcitool dev*** to find that my default bluetooth hci number is one.          
After searching, I found that Bleak use hci0 as default bluetooth setting, and it seems to NOT be set manually in Bleak.
Could you add a function that allow people to select specific hci number for additional BT dongle connecting?

**Comment by dlech** [comment, marked as answer]
There is already and `adapter` keyword argument that allows specifying a different adapter. Also related: #513.

**Comment by Ether-YiTseWu** [comment]
OK, thank you. I successfully selected the BT adapter manually.
